gauge_reader:
  ros__parameters:
    image_topic: "/camera/image_raw"
    use_math: false
    min_gauge_score: 0.99
    min_needle_score: 0.95
    scaling_min: 0.0
    scaling_max: 10.0
